src/Uno.UI/UI/Xaml/Controls/Primitives/LoopingSelector/LoopingSelectorAutomationPeer_Partial.cs

```diff
@@ -91,6 +91,130 @@ internal void RealizeItemAtIndex(int index)
 			}
 		}
 
+#if false
+		#region IItemsContainerProvider
```

Comment:
Where is this part coming from ?

Reply:
@jeromelaban What do you mean?

Reply:
@jeromelaban In case you are reviewing single commits. There was a code that I deleted but added back with `#if false`

Reply:
That was the case :) Thanks!